Gemfile.lock

```diff
@@ -1203,4 +1203,4 @@ RUBY VERSION
    ruby 3.2.2p53
 
 BUNDLED WITH
-   2.4.10
+   2.4.10
```

Comment:
Gemfile.lock should be removed from this branch.

Reply:
Ya working on it now, thanks for the heads up!

Reply:
ok after three tries this should be cleaned up

Reply:
Still seeing this file in your branch/github list of changed files.